demo/molecule/radioButtonGroup/demo/index.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+import React from 'react'
+
+import AtomRadioButton from '@s-ui/react-atom-radio-button'
+import AtomRadioButtonField from '@s-ui/react-molecule-radio-button-field'
+import AtomRadioButtonGroup from '../../../../components/molecule/radioButtonGroup/src'
```

Comment:
I know this is just for demo purposes but I guess these two components should be named `MoleculeRadioButtonWhatever`.